docs/knowledgebase/getting-started/glossary.md

```diff
@@ -2,32 +2,32 @@
 title: Glossary
 ---
 
-Glossary (alphabetical) of terms and lingo used in the Substrate and Polkadot codebases, as well as
-architecture.
+This page defines and explains many of the domain-specific terms that are common to the Substrate
+ecosystem. This is a helpful resource for even the most experienced Substrate developers.
 
-## Author (aka Block Author, Block Producer)
-
-The node, actor or identity that is responsible for the creation of a block. Synonymous with "miner"
-or "coinbase" on legacy PoW chains.
+---
 
 ## Adaptive Quorum Biasing (AQB)
 
-A means of specifying the quorum needed for a proposal to pass dependent upon voter turnout.
-Positive bias means requiring increasingly more aye votes than nay votes as there is lower turnout;
-negative (aka inverted) bias is the opposite (see the
+A means of specifying the quorum (minimum vote) needed for a proposal to pass dependent upon voter
+turnout. Positive bias means requiring increasingly more aye votes than nay votes as there is lower
+turnout; negative (aka inverted) bias is the opposite (see the
 [graphs](https://youtu.be/VsZuDJMmVPY?t=25413)). It avoids the need for strict quorums, which are
-arbitrary and bring about undesirable mechanics.
+arbitrary and bring about undesirable mechanics. Adaptive quorum biasing is used by the
+[FRAME Collective pallet](../runtime/frame#collective) that materializes as a governing
+[Council](#council) on a number of Substrate-based chains.
```

Comment:
1. AQB does not specify a quorum, it specifies a passing threshold _based on_ the quorum.
2. Collective does not use AQB, Democracy does.

Reply:
I updated this but I'm almost certain it doesn't make it "better". Can you help me improve it?